docs/use-linea/explore/use-patch.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Use Patch Wallet
+sidebar_position: 1
+---
+
+# Patch Wallet
+[Patch Wallet](https://linea.patchwallet.com/) is a "zero onboarding" wallet attached to every Twitter account, email, and telephone number. There's no app to download, no gas, or even sign-up. Every Twitter user has a wallet automatically and can start receiving tokens and NFTs!
+
+In this guide, we'll walk you through how to access your Patch wallet connected to your Twitter, mint an NFT, and send it to *any* Twitter user...[Elon Musk](https://linea.patchwallet.com/twitter:elonmusk), [Cristiano Ronaldo](https://linea.patchwallet.com/twitter:cristiano), [Narendra Modi](https://linea.patchwallet.com/twitter:narendramodi) or anyone else!
```

Comment:
<img width="936" alt="Screenshot 2023-06-13 at 6 06 40 PM" src="https://github.com/ConsenSys/doc.zk-evm/assets/17280653/b9bfb2c9-28d0-414a-ab2e-3e786a669ee9">

Hi can you add the failing words to `project-words.txt` (in alphabetical order at the top - I'm aware the bottom words are all out of order)
And run `npm run lint` to make sure they all pass

Reply:
Ok, added @emilyJLin95 👍

Let me know if you need any other changes.